models/Post.js

```diff
@@ -0,0 +1,21 @@
+// This is a code snippet, not used for production.
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const postSchema = new Schema({
+  title: String,
+  postedBy: {
+      type: mongoose.Schema.Types.ObjectId,
```

Comment:
After reading the link above I think you guys are right and ObjectId's will be better in the long run.